src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -25,6 +25,9 @@ public string Domain
         [SupportedOSPlatform("windows")]
         public bool LoadUserProfile { get; set; }
 
+        [SupportedOSPlatform("windows")]
```

Comment:
Note that the original issue said to determine the Windows version that the flag first became available in. I haven't found a version history of winbase.h, and [CreateProcessWithLogonW ](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createprocesswithlogonw) only says that the method was added in Windows XP/Windows Server 2003, but doesn't specify if the `LOGON_NETCREDENTIALS_ONLY` logon flag might have been added afterwards. However, [this forum post from 2030](https://forums.codeguru.com/showthread.php?228109-How-to-use-CreateProcessWithLogonW) did already refer to to the `LOGON_NETCREDENTIALS_ONLY` flag. Therefore, based on the OS support of .NET Core, just checking for `"windows"` *should*  be fine.

Reply:
Sounds reasonable to me. The oldest Windows we support is 7 and we have CI leg for it. So in case our assumptions turn out to be wrong, the test is going to fail.